Sources/Views/BatchUpdatesFromArrayDifference.swift

```diff
@@ -0,0 +1,57 @@
+extension BatchUpdates {
+    static func calculate(old: [[Any]], new: [[Any]], elementCompareCallback: (Any, Any) -> Bool) -> BatchUpdates {
```

Comment:
calculate is not a great name for method, old and new are not great names for variables, and `any` type should be avoided.